File: app/src/main/java/com/wxm/camerajob/ui/Base/SendEmailPara.java
package com.wxm.camerajob.ui.Base;

/**
 * parameter for send email
 * Created by 123 on 2016/11/10.
 */
public class SendEmailPara {
    public interface onSendEmailResult  {
        void onSendFailure();
        void onSendSuccess();
    }

    public onSendEmailResult    mIFOnResult;

    public String   mSendUsr;
    public String   mSendPWD;
    public String   mRecvUsr;
    public String   mSendServerType;
    public String   mSendServerHost;

    public String   mEmailTitle;
    public String   mEmailBody;
}

File: app/src/main/java/com/wxm/camerajob/ui/base/SendEmailPara.kt
package com.wxm.camerajob.ui.base

/**
 * parameter for send email
 * Created by WangXM on 2016/11/10.
 */
class SendEmailPara {
    var mIFOnResult: SendEmailResult? = null

    var mSendUsr: String? = null
    var mSendPWD: String? = null
    var mRecvUsr: String? = null
    var mSendServerType: String? = null
    var mSendServerHost: String? = null

    var mEmailTitle: String? = null
    var mEmailBody: String? = null

    interface SendEmailResult {
        fun onSendFailure()
        fun onSendSuccess()
    }
}



